gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/sampling/statistics/AverageDegree.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.model.impl.operators.sampling.statistics;
+
+import org.apache.flink.api.java.DataSet;
+import org.gradoop.common.model.impl.pojo.GraphHead;
+import org.gradoop.flink.model.api.epgm.LogicalGraph;
+import org.gradoop.flink.model.api.operators.UnaryGraphToGraphOperator;
+import org.gradoop.flink.model.impl.operators.aggregation.functions.count.VertexCount;
+import org.gradoop.flink.model.impl.operators.sampling.statistics.functions.AddSumDegreesToGraphHeadCrossFunction;
+import org.gradoop.flink.model.impl.operators.sampling.statistics.functions.CalculateAverageDegree;
+import org.gradoop.flink.model.impl.operators.statistics.VertexDegrees;
+
+/**
+ * Calculates the average degree of a graph and writes it to the graph head.
+ * Uses: {@code ceiling( sum(vertex degrees) / |vertices| )}
+ */
+public class AverageDegree implements UnaryGraphToGraphOperator {
+
+  /**
+   * {@inheritDoc}
```

Comment:
Inheriting here results in a doc block 
```
/**
   * Executes the operator.
   *
   * @param graph input graph
   * @return operator result
   */
```
which is not meaningful. Same for other degree classes and their execute function.